spec/features/csrf_error_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe 'CSRF errors' do
+  it 'does not show any errors' do
+    visit idp_reset_password_url
+
+    expect(page).to have_content('Forgot')
+
+    fill_in 'password_reset_email_form_email', with: 'fake-email@login.gov'
+
+    page.driver.browser.remove_cookie 'AWSALB'
```

Comment:
@monfresh we're only deleting the AWS ALB cookie, nothing at the app level

If the app behaved as expected, the session data would not be tied to a host or process, and the page would submit succesfully

Reply:
Ah. I see now. When y'all were mentioning the AWS ALB cookie, I wasn't sure what you meant. I see that there's an actual cookie called AWSALB. Where does that come from? Why is our app setting that cookie?

Reply:
Never mind. I know the answer. I'll read up on AWS sticky sessions.